apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,71 +17,51 @@ import * as AllExamples from 'examples';
-### Informational
+### Composition API
 
-Default style when there is no defined status for the alert. Inform users about events that they should be aware of, but that are not disruptive to their work.
+You can build Alerts with subcomponents that are fully customizable.
 
-<LiveExample src='Alert.informational.tsx'>
-  <AllExamples.AlertInformationalExample client:load />
-</LiveExample>
-
-### Positive
-
-The positive alert is used to inform the user of a positive status.
-
-<LiveExample src='Alert.positive.tsx'>
-  <AllExamples.AlertPositiveExample client:load />
-</LiveExample>
-
-### Warning
+We provide:
 
-The warning alert is used to inform the user of something that is not currently disruptive to their work, but may be soon or eventually.
+- Alert.Wrapper
+- Alert.Icon
+- Alert.Message
+- Alert.Action
+- Alert.CloseButton
 
-<LiveExample src='Alert.warning.tsx'>
-  <AllExamples.AlertWarningExample client:load />
-</LiveExample>
-
-### Negative
+The type of alert can be specified by utilizing the `type` prop inside `<Alert.Wrapper>`. There are four distinct types of alerts that can be chosen from, namely positive, negative, warning, or informational. Positive alerts are an indication that a task or action has been successfully completed, while warning alerts require attention but do not result in any immediate disruption. Negative alerts, however, signify system outages or failures. Informational alerts are intended to provide users with updates on non-disruptive tasks or work.
 
-Use this only when the system is broken. Leave the technical details behind and explain the problem to the user in an easily understood tone. Mention when the problem might get solved, if it applies.
+By passing the `onClick` prop to `<Alert.Action>` or `<Alert.CloseButton>`, the user can interact with the alert to show additional information or to close. A console message can be logged for event response.
 
-<LiveExample src='Alert.negative.tsx'>
-  <AllExamples.AlertNegativeExample client:load />
+<LiveExample src='Alert.types.tsx'>
+  <AllExamples.AlertTypesExample client:load />
 </LiveExample>
 
-### Multilines
-
-All alerts may support multiple lines if necessary. We recommend keeping it brief and keep the message as concise as possible.
-
-## Placement
+## Variants
 
-The alert notification can display in two different ways:
+### Sticky Alert
 
-### Inline
-
-An inline alert appears embedded within the copy of the site as a block element. This means that when the user scrolls, the alert scrolls as well.
+An alert can be fixed at the top of a user's screen, even if they scroll down the page by passing the `isSticky` prop to the `<Alert.Wrapper>` component. It's important to note that sticky alerts must always be dismissible. Additionally, the alert will appear above all other UI elements except for toast notifications and modals.
 
-<LiveExample src='Alert.inline.tsx'>
-  <AllExamples.AlertInlineExample client:load />
+<LiveExample src='Alert.sticky.tsx'>
```

Comment:
The sticky example can be modified to better show that the Alert sticks to the top. You can create a tall scrollable page with a small `max-height` (or something relative like `80%` might work too) and `overflow: auto`, similar to the [Alert -> Sticky story](https://itwin.github.io/iTwinUI/react/?path=/story/core-alert--sticky). Then when the user scrolls, they can see how the Alert sticks to the top.

Reply:
I tried writing modifying the selector of the scrollbar styles to exclude everything in the iTwinUI scope (descendants of `._iui3-root`):
![image](https://github.com/iTwin/iTwinUI/assets/45748283/efc810c6-fa13-4597-b2fe-50ca75b0825c)

But that unfortunately didn't work as I expected.

After speaking with @mayank99 , he suggested to remove all scrollbar customizations completely. So with #1673, this will be solved.